### ISSUE TITLE
Ischemic priapism

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10100]
**Preferred term label:**
Ischemic priapism

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C2711256

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Ischemic priapism

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
the HPO typically does not include etiology (ischemic) and so probably it would be best to just code https://hpo.jax.org/app/browse/term/HP:0200023 unless there are typical phenotypic manifestations ?

**Comment by MickeySegal** [comment]
I only used the term "Ischemic priapism" for UMLS compatibility, but what we call the finding is "Penile (corporal) blood gas ischemic", which fits better with the HPO approach.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000625
- label: Penile corporal blood gas ischemia
- parents: [Abnormal penis morphology (HP:0000036)](https://hpo.jax.org/app/browse/term/HP:0000036)
- pmid: PMID:38496140
- def: Partial pressure of oxygen (pO2) in cavernous blood of the penis is below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10100
- synonyms:
